apps/desktop/src/renderer/src/app/styles/theme.css

```
@@ -4,8 +4,9 @@
   --radius-*: initial;
   --font-*: initial;
 
-  --font-sans: -apple-system, 'SF Pro Text', system-ui, sans-serif;
-  --font-mono: 'SF Mono', ui-monospace, Menlo, monospace;
+  --font-sans:
+    -apple-system, 'SF Pro Text', 'Segoe UI Variable Text', 'Segoe UI', system-ui, sans-serif;
+  --font-mono: 'SF Mono', ui-monospace, 'Cascadia Mono', Consolas, Menlo, monospace;
```

Stylelint (17.14.0)

[error] Expected "Consolas" to be "consolas" (value-keyword-case)

(value-keyword-case)